src/graphdb.js

```diff
@@ -147,39 +186,75 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
         r += '}\n\n';
 
         // input for the node type
-        r += `input ${nodeCase}Input {\n`;
+        let nodeCase2 = replaceCleanseLabel(node.label);
+
+        if (node.label !== nodeCase2) {
+            r += `input ${toPascalCase(nodeCase2)}Input @alias(property: "${node.label}") {\n`;
```

Comment:
Hmm I don't think alias for inputs are used by the resolver. When I remove this logic to add the alias the query still works:
![Screenshot 2024-11-07 at 3 55 44 PM](https://github.com/user-attachments/assets/0b54be47-2490-403f-893e-2a1b5987a42f)